framework/src/Volo.Abp.Cli.Core/Volo.Abp.Cli.Core.csproj

```diff
@@ -31,4 +31,8 @@
     <ProjectReference Include="..\Volo.Abp.Json\Volo.Abp.Json.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="Volo\Abp\Cli\Utils\ConfigReaderHelper.cs" />
```

Comment:
I suppose this is by accident, we can remove.

Reply:
Thanks, removed.